firefox-ios/Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift

```diff
@@ -17,10 +17,12 @@ protocol PhotonActionSheetProtocol {
 extension PhotonActionSheetProtocol {
     typealias PresentableVC = UIViewController & UIPopoverPresentationControllerDelegate
 
-    func presentSheetWith(viewModel: PhotonActionSheetViewModel,
-                          on viewController: PresentableVC,
-                          from view: UIView) {
-        // TODO: Regression testing needed here.
+    func presentSheetWith(
+        viewModel: PhotonActionSheetViewModel,
+        on viewController: PresentableVC,
+        from view: UIView,
+        with uuid: WindowUUID
```

Comment:
@thatswinnie ok. But I don't understand... why do we need the UUID in the function call? if we have `button`, we just do the same thing on line 26?

Reply:
We don't. The button returned nil as the uuid before but that was related to how the toolbars were updated (the button was not in the view stack anymore). Now that this is fixed we don't need the uuid any longer. 

So we can basically close the PR, correct?